src/iuse.cpp

```diff
@@ -6910,6 +6905,65 @@ int iuse::holster_ankle(player *p, item *it, bool b, const tripoint &pos)
     return it->type->charges_to_use();
 }
 
+int iuse::belt_loop (player *p, item *it, bool, const tripoint&)
+{
+    // if belt loop empty offer to attach an item otherwise detach the current one
+    if (it->contents.empty()) {
+        // display menu showing only show BELT_CLIP items
+        item *put = &(p->i_at(g->inv_for_flag("BELT_CLIP", _("Attach what to belt loop?"), false)));
+        if (put == NULL || put->is_null()) {
```

Comment:
A similar idiom is used elsewhere in iuse.cpp. Is it appropriate to correct those as well in the same PR?